<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMain</class>
 <widget class="QMainWindow" name="FormMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EPICS Archiver Appliance GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listBuffer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>360</width>
      <height>361</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNow">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>450</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Now</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>450</y>
      <width>17</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>To</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dtTo">
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>450</y>
      <width>202</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="displayFormat">
     <string>dd/MM/yyyy hh:mm:ss AP</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="listData">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>360</width>
      <height>360</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>780</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QComboBox" name="cbMethod">
    <property name="geometry">
     <rect>
      <x>703</x>
      <y>450</y>
      <width>87</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <item>
     <property name="text">
      <string>Standard</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mean</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>min</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>max</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>median</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>std</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>jitter</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>variance</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>34</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>From</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dtFrom">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>450</y>
      <width>202</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="displayFormat">
     <string>dd/MM/yyyy hh:mm:ss AP</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>629</x>
      <y>450</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Processing</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>40</width>
      <height>140</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QPushButton" name="btnAdd">
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRemove">
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAddAll">
     <property name="text">
      <string>&gt;&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRemoveAll">
     <property name="text">
      <string>&lt;&lt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>780</width>
      <height>30</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtSearch">
     <property name="statusTip">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Template Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtName">
     <property name="statusTip">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>780</width>
      <height>30</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QRadioButton" name="rbWildcard">
     <property name="text">
      <string>Wildcard</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="rbStartsWith">
     <property name="text">
      <string>Starts with</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="rbEndsWith">
     <property name="text">
      <string>Ends with</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="rbRegex">
     <property name="text">
      <string>Regex</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QCheckBox" name="cbDefault">
     <property name="text">
      <string>Default Directory</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>780</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Save Template</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLoad">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Load Template</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPlotData">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Plot Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExportCSV">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Export CSV</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExportMAT">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Export MAT</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
